conf/deployment/aws/ipi_3az_rhcos_3m_6w_arbiter.yaml

```diff
@@ -0,0 +1,25 @@
+---
+#  This configuration file is used for Metro DR Arbiter cluster deployment
+#  TODO: Add polarion ID
+DEPLOYMENT:
+  arbiter_deployment: true
+  allow_lower_instance_requirements: false
+  type: 'EBS'
+  local_storage: true
+  ocs_operator_nodes_to_label: 6
+  arbiter_zone: 'us-east-2c'
+  arbiter_autodetect: false
```

Comment:
I would also add `network_split_setup: true` so that it's possible to try net splits if necessary.